zenfolio.com.customdomain.json

```diff
@@ -0,0 +1,41 @@
+{
+    "providerId": "Zenfolio",
+    "providerName": "Zenfolio Custom Domain Web Hosting",
+    "serviceId": "CustomDomain",
```

Comment:
```AssertionError: File name zenfolio.com.customdomain.json not matching the pattern. Expected zenfolio.customdomain.json
File name zenfolio.com.customsubdomain.json not matching the pattern. Expected zenfolio.customsubdomain.json```

Reply:
Please continue your review and submit ALL requested changes at the same time so I can be more efficient with the revisions to save both of us time.

Reply:
Sure, these errors are detected automatically by CI script. You should have received a notification about failed build as well.
For some reason Travis didn't run on the initial PR, can't help here.

Reply:
For whatever reason, I did not get any notifications about the failed build.  I saw that I cannot merge into master (as expected), but nothing about CI/CD failures.  If the filenames are the last of it, I'll have the changes updates in a few minutes.